### PR TITLE
fix issue and add test

### PR DESCRIPTION
**Pull request by HendrikKok**

### CODE REVIEW

File: tests/test_imod_coupler/test_ribametamod_cases.py [hunk 1]
```diff
@@ -326,6 +328,59 @@ def case_bucket_model(
     )
 
 
+def case_bucket_model_no_subgrid(
```
Comment (rleander) [inline tests/test_imod_coupler/test_ribametamod_cases.py:331]: Is the case of basins coupled to metaswap but not to modflow also tested here  ?

**Review by rleander — CHANGES_REQUESTED** [review]

File: tests/fixtures/fixture_ribasim.py [hunk 2]
```diff
@@ -36,6 +36,13 @@ def ribasim_bucket_model() -> ribasim.Model:
     return add_subgrid(bucket)
 
 
+@pytest_cases.fixture(scope="function")
+def ribasim_bucket_model_no_subrgid() -> ribasim.Model:
```
[the remark this answers is not in the document]
Comment (HendrikKok) [reply tests/fixtures/fixture_ribasim.py]: done

File: tests/test_imod_coupler/test_ribametamod_cases.py (hunk 1 above)
Comment (HendrikKok) [reply tests/test_imod_coupler/test_ribametamod_cases.py:331]: No that is indeed not tested for now.

**Review by rleander — APPROVED** [review]
Prima werk!

**Pull request merged**